### DISCUSSION TITLE
Overlapping buckets

### DISCUSSION THREAD

**Discussion by latenssi** [flow-hydraulics/flow-pds#3] [General]
Can buckets overlap? As in can bucket 1 contain same IDs as bucket 2?

I think we should avoid this and deal with this outside the PDS.

**Comment by aphelionz** [comment]
I would assume that they do not overlap i.e. "rare" NFTs cannot also be "common" NFTs. Is this what you mean?

**Comment by EricLin2004** [comment]
Bucket 1 and Bucket 2 cannot have the same NFTs in them.

**Reply by latenssi** [reply]
Check!
